### PR TITLE
[T3A2][T16-A4]Bennett Goh

### PR DESCRIPTION
**Pull request by bennettgo**

### CODE REVIEW

**Comment by bennettgo** [comment]
Ready for review

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -11,16 +11,17 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private final String feedbackToUser;
```
Comment (sooyj) [inline src/seedu/addressbook/commands/CommandResult.java:14]: Good work. Field changed to private.

File: src/seedu/addressbook/commands/CommandResult.java [hunk 2]
```diff
@@ -32,5 +33,8 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
     public Optional<List<? extends ReadOnlyPerson>> getRelevantPersons() {
         return Optional.ofNullable(relevantPersons);
     }
-
+    
+    public String getFeedbackToUser(){
```
Comment (sooyj) [inline src/seedu/addressbook/commands/CommandResult.java:37]: Correct! You now need a getter method for outsider.

File: src/seedu/addressbook/data/person/Block.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
+public class Block {
```
Comment (sooyj) [inline src/seedu/addressbook/data/person/Block.java:3]: Seems you have A3 changes mixed in this branch?

File: src/seedu/addressbook/ui/TextUi.java [hunk 4]
```diff
@@ -125,7 +125,7 @@ public void showResultToUser(CommandResult result) {
         if (resultPersons.isPresent()) {
             showPersonListView(resultPersons.get());
         }
-        showToUser(result.feedbackToUser, DIVIDER);
+        showToUser(result.getFeedbackToUser(), DIVIDER);
```
Comment (sooyj) [inline src/seedu/addressbook/ui/TextUi.java:128]: Correct! All outsider references need to go through the getter method to access information.

**Review by sooyj — COMMENTED** [review]
Good attempt. Field encapsulated, getter method added and all references updated.

However, there are some changes meant for A3 mixed in this branch.

**Pull request closed without merging**